src/admin-views/calendar-embeds/embed-snippet-content.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Content for the Embed Snippet column within the Calendar Embeds List Table.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ * @var int    $post_id   The post ID.
+ * @var string $permalink The permalink.
+ */
+
+// @todo Maybe passed the snippet in as a variable instead of generating it here?
+$snippet = '<iframe src="' . esc_url( $permalink ) . '" width="100%" height="600" frameborder="0"></iframe>';
+
+?>
+<div id="tec_events_calendar_embeds_snippet_<?php echo esc_attr( $post_id ); ?>" class="hidden">
+	<div>
+		<div class="tec-events-calendar-embeds__snippet-modal-text">
+			<?php esc_html_e( 'Copy and paste this code to embed the calendar on your website:', 'the-events-calendar' ); ?>
+		</div>
+		<textarea
+			id="tec_events_calendar_embeds_snippet_code_<?php echo esc_attr( $post_id ); ?>"
+			class="tec-events-calendar-embeds__snippet-modal-textarea"
+			aria-label="<?php esc_attr_e( 'Embed snippet code', 'the-events-calendar' ); ?>"
+			rows="3"
+			readonly><?php echo esc_html( $snippet ); ?></textarea>
```

Comment:
Note: this is a textarea, you can (maybe should?) use [esc_textarea](https://developer.wordpress.org/reference/functions/esc_textarea/)